crates/db_views/src/post_view.rs

```diff
@@ -196,6 +197,13 @@ fn queries<'a>() -> Queries<
     let local_user_id_join = local_user_id.unwrap_or(LocalUserId(-1));
 
     let mut query = all_joins(post_aggregates::table.into_boxed(), person_id)
+      .left_join(
+        instance_block::table.on(
+          post_aggregates::instance_id
```

Comment:
Since the other blocks all use columns directly on the post_aggregates table (pa::creator_id, pa::community_id), I think you made the right decision then to add the `instance_id` column to it, and the trigger below. Its a filtered column, so it makes sense for performance reasons.